server/gql/schema.js

```diff
@@ -56,6 +75,37 @@ const listDataSources = ({ name }) => {
     return dataSource.findAll();
 };
 
+const listResolvers = ({ schemaId, type }) => resolver.findAll({
+    where: {
+        GraphQLSchemaId: schemaId,
+        type
+    },
+    include: {
+        model: dataSource,
+        as: "DataSource"
+    }
+});
+
+const upsertResolver = async ({
+    id, schemaId, dataSourceId, type, field,
+    requestMapping, responseMapping
+}) => {
+    const properties = {
+        GraphQLSchemaId: schemaId,
+        DataSourceId: dataSourceId,
```

Comment:
Is `properties.DataSourceId` upper case on purpose?

Reply:
Yes, thats how Sequelize expects them.

Reply:
Could be overridden i guess. There is also a different API that we can use to add a resolver to a schema and data source but that would require us to fetch those instances first.